tox.ini

```diff
@@ -0,0 +1,44 @@
+[tox]
+envlist =
+    py{37,38}-orange-{oldest, latest, released}
+    future
+skip_missing_interpreters = true
+isolated_build = true
+
+[testenv]
+# must use latest pip (version 20.3.1 enables Big Sur support - https://github.com/pypa/pip/issues/9138)
+pip_version = pip
+extras = test
+passenv = *
+# we MUST changedir to avoid installed being shadowed by working dir
+# https://github.com/tox-dev/tox/issues/54
+# https://github.com/tox-dev/tox/issues/514
+changedir =
+    {envsitepackagesdir}
+setenv =
+    # Raise deprecations as errors in our tests
+    ORANGE_DEPRECATIONS_ERROR=y
+    # Need this otherwise unittest installs a warning filter that overrides
+    # our desire to have OrangeDeprecationWarnings raised
+    PYTHONWARNINGS=module
+    # set coverage output and project config
+    COVERAGE_FILE = {toxinidir}/.coverage
+    COVERAGE_RCFILE = {toxinidir}/.coveragerc
+deps =
+    pyqt5==5.12.*
+    pyqtwebengine==5.12.*
+    oldest: orange3==3.25.0
+    oldest: orange-canvas-core==0.1.15
```

Comment:
Please move to oldest dependencies that still pass, and comment where these are not the same as for the then-released Orange